src/components/listcase.ts

```diff
@@ -49,16 +49,9 @@ export async function run(
             .setDisabled(page + 1 === chunk.length)
     );
 
-    const footer = client.i18n.t('LISTCASE_FOOTER', {
-        lng: interaction.locale
-    });
-
     const embed = new Embed(client, interaction.locale)
         .setLocaleTitle('LISTCASE_TITLE', { user: member.tag })
-        .setThumbnail(member.displayAvatarURL({ dynamic: true }))
-        .setFooter({
-            text: `${footer} | ${client.footer}`
-        });
+        .setThumbnail(member.displayAvatarURL({ dynamic: true }));
```

Comment:
why remove the footer

Reply:
because client footer litreally is no moer